tests/scripting_api/test_engine.py

```diff
@@ -29,9 +31,14 @@
 import autokey.service
 from autokey.scripting import Engine
 
+# For use in paramerizations.
+# Call replace_folder_in_args(folder, args) to use.
+folder_param="create_new_folder"
+
 def get_autokey_dir():
     return os.path.dirname(os.path.realpath(sys.argv[0]))
 
+@pytest.fixture
```

Comment:
Ah. _That’s how_ you use the decorator. I’ve never figured out myself. TIL.

Reply:
I fumbled around for a bit until things passed